tests/python/relay/aot/test_crt_forward_declarations.py

```diff
@@ -0,0 +1,325 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""test forward function declarations codegen by CodegenCHost."""
+
+from collections import OrderedDict
+import pytest
+import numpy as np
+
+import tvm.testing
+from tvm import relay
+from tvm.contrib.download import download_testdata
+from tvm.relay.op.contrib import cmsisnn
+from tvm.testing.aot import AOTTestModel, compile_models, generate_ref_data
+from tvm.micro.testing.aot_test_utils import (
+    AOT_CORSTONE300_RUNNER,
+    AOT_USMP_CORSTONE300_RUNNER,
+    parametrize_aot_options,
+    AOTTestRunner,
+)
+
+
+def skip_if_no_reference_system(func):
+    return tvm.testing.skip_if_32bit(reason="Reference system unavailable in i386 container")(func)
+
+
+def get_range_for_dtype_str(dtype):
```

Comment:
This function is only used once and with a fixed dtype so it doesn't feel necessary to include it just for that, could we either move it to a testing util or just hardcode the range?

Reply:
If we can leave it here for now, it will be easier to find it by name when its cleaned up. A similar function is being used by a lot of other tests. I plan to clean it up soon for Arm targets :smile:

Reply:
Ok dokie 😸